scripts/docgen/snippets/extractSnippets.ts

```diff
@@ -81,15 +82,15 @@ const extractLabeledSnippets = (sourceFileText: string): LabeledSnippets => {
     const labeledSnippets: Record<string, Snippet> = {}
     const openBlocks: SnipStart[] = []
 
-    const textSplitByNewLine = sourceFileText.split("\n")
+    const lines = sourceFileText.split("\n")
     let text
```

Comment:
In general, variables that are used within a loop should be defined within the loop itself to clarify their scope (there should be no reason to use `text` or `lineNumber` outside the body of the loop.